src/constructs/autoscaling/asg.ts

```diff
@@ -162,5 +162,7 @@ export class GuAutoScalingGroup extends GuAppAwareConstruct(AutoScalingGroup) {
     // leaves it to the default value, which is actually false.
     // { UpdatePolicy: { autoScalingScheduledAction: { IgnoreUnmodifiedGroupSizeProperties: true }}
     cfnAsg.addDeletionOverride("UpdatePolicy");
+
+    Tags.of(launchTemplate).add("App", app);
```

Comment:
Do we want to make the `app` prop required? I think its a `string | undefined` at the moment, could lead to the App tag being `undefined`

Reply:
Ha! You've found what I'm doing this 10% day 😄!

It's required on the EC2 App pattern, so it will be `string` here. It's `string | undefined` at the `GuStack` level.